LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -93,6 +98,25 @@ function openContent(url) {
         	cache: false
         });
 
+        // load group data
+        dialog.indicateBusy();
+        $.ajax({
+            type: 'GET',
+            url: '../api/?a=group/members',
+            success: function(res) {
+                if(res.length > 0) {
+                    for(var i in res) {
+                        $('#cloneGroupID').append('<option value="'+ res[i].groupID +'">'+ res[i].name +'</input>');
```

Comment:
The `sysadmin` group needs to be filtered out of these results, it will always have a `groupID` of `1`. Sysadmins can clone any form on their RP.

Reply:
added conditional in commit 96c excluding sysAdmin